Formula/fping.rb

```diff
@@ -14,20 +14,28 @@ class Fping < Formula
   end
 
   head do
-    url "https://github.com/schweikert/fping.git"
+    url "https://github.com/schweikert/fping.git", branch: "develop"
+
     depends_on "autoconf" => :build
     depends_on "automake" => :build
   end
 
   def install
     system "./autogen.sh" if build.head?
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--sbindir=#{bin}"
+    system "./configure", *std_configure_args, "--sbindir=#{bin}"
     system "make", "install"
   end
 
   test do
-    assert_equal "::1 is alive", shell_output("#{bin}/fping -A localhost").chomp
+    assert_match "Version #{version}", shell_output("#{bin}/fping --version")
+    assert_match "Probing options:", shell_output("#{bin}/fping --help")
+
+    can_run_fping = true
+    on_linux { can_run_fping = Process.uid.zero? }
```

Comment:
We don't run as root in CI, so I'm not sure we need this.  Did you add this because the local Docker is still running as root?  This is something that should eventually be fixed.

Reply:
Mainly in case someone wants to check test passes locally for Linux. Not essential and can remove if preferred.

Reply:
I'd rather we avoid workarounds for someone running as root, which is something we actively discourage for some very good reasons.